res/values-pt-rBR/string.xml

```diff
@@ -0,0 +1,160 @@
+    <string name="system_app_remover">Remover apps do sistema</string>
+    <string name="dialer_title">Dialer</string>
+
+    <!-- Color Picker -->
+    <string name="dialog_color_picker">Seleção de cor</string>
+    <string name="press_color_to_apply">Pressione uma cor abaixo para aplicar</string>
+    <string name="arrow_right">→</string>
+    <string name="arrow_down">↓</string>
+    <string name="hex">Hex:</string>
+    <string name="hex_hint">#ff000000</string>
+    <string name="set">Definir</string>
+    <string name="color_default">Padrão</string>
+
+    <!-- Hardware keys -->
+    <string name="hardware_keys_category">Hardware keys</string>
+    <!-- Buttons backlight timeout -->
+    <string name="backlight_timeout_title">Buttons backlight timeout</string>
+    <string name="backlight_timeout_on">Sempre ativo</string>
+    <string name="backlight_timeout_3s">3 segundos</string>
+    <string name="backlight_timeout_default">5 segundos (Padrão)</string>
+    <string name="backlight_timeout_10s">10 segundos</string>
+    <string name="backlight_timeout_15s">15 segundos</string>
+    <string name="backlight_timeout_20s">20 segundos</string>
+    <string name="backlight_timeout_30s">30 segundos</string>
+    <!-- Enable keys backlight-->
+    <string name="button_brightness_title">Backlight</string>
+
+    <!-- Power Menu -->
+    <string name="power_menu_title">Menu de Energia</string>
+    <string name="power_menu_title_summary">Ver ou mudar entradas do Menu de Energia</string>
+    <string name="powermenu_screenshot">Captura de Tela</string>
+    <string name="powermenu_screenrecord">Gravação de Tela</string>
+    <string name="powermenu_power">Desligar</string>
+    <string name="powermenu_restart">Reiniciar</string>
+    <string name="powermenu_restart_recovery">Reinicialização avançada</string>
+    <string name="powermenu_airplane">Modo Avião</string>
+    <string name="powermenu_settings">Configurações</string>
+    <string name="powermenu_lockdown">Bloquear</string>
+    <string name="powermenu_soundpanel">Painel de Som</string>
+    <string name="powermenu_users">Usuários</string>
+    <string name="powermenu_logout">Logout</string>
+
+    <!-- HW Keys customization -->
+    <string name="action_value_default_action" translatable="false">Padrão</string>
+    <string name="action_value_select_app" translatable="false">app</string>
+    <string name="action_value_custom_action" translatable="false">custom</string>
+    <string name="action_value_back" translatable="false">task_back</string>
+    <string name="action_value_home" translatable="false">task_home</string>
+    <string name="action_value_search_assist" translatable="false">task_assist</string>
+    <string name="action_value_recent_apps" translatable="false">task_recents</string>
+    <string name="action_value_voice_search" translatable="false">task_voice_search</string>
+    <string name="action_value_menu" translatable="false">task_menu</string>
```

Comment:
@joaoeudes7 do not need to add to the localization of the line with translatable="false"

Reply:
Ok, removed